src/tests/test_data_service.cpp

```diff
@@ -56,13 +60,15 @@ constexpr uint64_t Mi{Ki * Ki};
 constexpr uint64_t Gi{Ki * Mi};
 
 struct Param {
-    uint64_t num_io;
-    uint64_t run_time;
+    uint64_t num_io{1000};
+    uint64_t run_time{200};        // in seconds
+    uint32_t min_io_size{4 * Ki};  // blk_size aligned;
+    uint32_t max_io_size{32 * Mi}; // blk_size aligned;
 };
 
 static Param gp;
 
-ENUM(DataSvcOp, uint8_t, WRITE, READ, FREE_BLK, COMMIT_BLK, RESERVE_STREAM, ALLOC_STREAM, FREE_STREAM)
+ENUM(DataSvcOp_t, uint8_t, async_alloc_write = 1, async_read = 2, async_free = 3, max_op = 4);
```

Comment:
If you want to allocate the value for it we should use `VENUM(..`. I don't see much benefit in assigning the value here.

Reply:
acked